src/config.js

```diff
@@ -57,7 +59,9 @@ module.exports = {
   linkTableBlockKey: 'blockmultihash',
   linkTableCarKey: 'carpath',
 
+  enableKeepAlive: enableKeepAlive === 'true',
```

Comment:
We should keep this consistent and easier to read

`enableKeepAlive: enableKeepAlive ?? 'true',`

Reply:
I'd like to release that and monitor for a day, then restore it
I think it's easier doing in this way, but happy with any solution